services/core-api/app/config.py

```diff
@@ -157,6 +159,13 @@ def JWT_ROLE_CALLBACK(jwt_dict):
     EMAIL_ENABLED = os.environ.get('EMAIL_ENABLED', False)
     EMAIL_RECIPIENT_OVERRIDE = os.environ.get('EMAIL_RECIPIENT_OVERRIDE')
 
+    #Templates
+    TEMPLATE_FOLDER_BASE = os.environ.get('TEMPLATE_FOLDER_BASE', '/app/app/templates')
```

Comment:
Is "/app/app" a typo here? I notice the templates folder is "/app/templates".

Reply:
relative path is considered.